src/main/cpp/chassis/states/DriveToFieldElement.cpp

```diff
@@ -89,6 +92,7 @@ std::array<frc::SwerveModuleState, 4> DriveToFieldElement::UpdateSwerveModuleSta
 
         chassisSpeeds.vx = pidVx;
         chassisSpeeds.vy = pidVy;
+        // Shouldn't we set Omega here; otherwise we are picking up what was previously set (maybe from trajectory)?
```

Comment:
We don't want to set the omega here because in the init we set our heading option to be FaceNearestReefFace or coral station, and the omega gets calculated and set previously in the SwerveChassis Drive method